api/controller/EducationController.py

```diff
@@ -12,15 +12,17 @@ def get_education(self, id):
             education = Education.objects.get(id=id)
             return {
                 "data": {
-                    "type": str(education),
-                    "id": str(education.id),
-                    "attributes": {
-                        "classification": education.classification,
-                        "question": education.question,
-                        "description": education.description,
-                        "information": education.information,
-                        "note": education.note,
-                        "source": education.source
+                    "01_type": str(education),
+                    "02_id": str(education.id),
+                    "03_attributes": {
+                        "A_order": education.order,
```

Comment:
I love the letter at the beginning of each key. what was the reasoning behind having the letter?

Reply:
@Atos20 This helped us force ordering. We first attempted to use Python's native `OrderedDict` and we tried to use the JSON native options, but both were unsuccessful so this was our current workaround. We are looking into going back to finding a way to natively Order this dictionary, but it is a lot of detailed work that we worried we wouldn't have enough time to accomplish this and roll out a successful production server for FE

Basically - it's currently a bandaid, but it's our best bandaid, and we can explain this more during our standup this morning!